### PR TITLE
Docs: Add page about setting raw HTML content (#2189)

### LINKED ISSUE #2189: Add page about setting raw HTML content
From time to time a user on discord runs into the issue that text content is always escaped which breaks script tag content. We should have a page in our docs about that which points to `dangerouslySetInnerHTML`.

### PR DESCRIPTION
**Pull request by CAYdenberg**
Closes #2189

### CODE REVIEW

**Comment by CAYdenberg** [comment]
Bump. Rebased to main.

File: docs/latest/examples/rendering-raw-html.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+result in **escaped characters, and will not work**:
+
+```tsx
+const json = `
+{
+  "@context": "http://schema.org",
+  "@type": "PostalAddress",
+  "streetAddress": "8888 University Drive",
+  "addressLocality": "Burnaby",
+  "addressRegion": "British Columbia"
```
Comment (marvinhagemeister) [inline docs/latest/examples/rendering-raw-html.md:32]: Nit: The json is missing the closing `}`

**Review by marvinhagemeister — APPROVED** [review]
Good idea! thanks for adding this 👍  Found one tiny nit, but that should be easy to fix

**Review by marvinhagemeister — APPROVED** [review]
Sweet, thanks 💯

**Pull request merged**